### PR TITLE
Feature/code search guide page [WIP]

### PR DESCRIPTION
**Pull request by mgyorffy**

### CODE REVIEW

**Comment by content-hacker** [comment]
FYI @jennybergen this is where we're building the code search page

File: website/src/pages/code-search-guide.tsx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+                    </div>
+
+		            <hr className="my-md-6" id="" />
+            	</div>
+
+            	<div className="col-lg-7 offset-lg-4" id="3">
+                    <h2 className="display-3">3 Why code search?</h2>
+                    <p> Once developers have code search they wonder how they ever lived without it. Being able to quickly find, understand, and fix the code you’re looking for just makes sense when 75% of a developer’s time is spent reading and understanding code. Not surprisingly, this sentiment is quite common and motivating among people who use and build code search tools. Han-Wen Nienhuys, the creator of the Zoekt code search engine, says:</p>
+                        
+                    <blockquote>
```
Comment (content-hacker) [inline website/src/pages/code-search-guide.tsx:112]: @mgyorffy here is the quote I need styled

**Comment by content-hacker** [comment]
Preview link: https://deploy-preview-2983--sourcegraph.netlify.app/code-search-guide/

File: website/src/pages/code-search-guide.tsx (hunk 1 above)
Comment (mgyorffy) [reply website/src/pages/code-search-guide.tsx]: @content-hacker added quote styling :)

File: website/src/pages/code-search-guide.tsx [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+                        help you navigate your code easier, no matter what tool(s) you may have access to.
+                    </p>
+                        
+                    <p>
+                        Our intention with this guide is to help every dev and development team navigate their codebase 
+                        with greater ease so you can spend more time creating and less time agonizing over unintended 
+                        rabbit holes. Let's get started!
+                    </p>
+
+                    <div className="container my-4 video-embed embed-responsive embed-responsive-16by9 ">
```
Comment (content-hacker) [inline website/src/pages/code-search-guide.tsx:109]: @mgyorffy is this suppose to be here? When I preview the page there is a huge, blank gap between the end of the first section and the start of section 2.

Comment (mgyorffy) [reply website/src/pages/code-search-guide.tsx:109]: @content-hacker I think you might just be experiencing the youtube video loading slowly. That is just temporary. This is what I see.
Screen Shot 2021-06-22 at 12 12 00 PM

Comment (content-hacker) [reply website/src/pages/code-search-guide.tsx:109]: Got it. These videos are not part of our guide. Can we remove them, or where did they come from?

Comment (mgyorffy) [reply website/src/pages/code-search-guide.tsx:109]: @content-hacker 
I think they were all in the original requirement doc. Should I remove all the videos?

**Comment by content-hacker** [comment]
@mgyorffy I added more content to this but I don't think I formatted headers, tables, and lists correctly. Can you take a look?

**Comment by mgyorffy** [comment]
@content-hacker all table-related issues are fixed now :) let me know if it looks good

**Pull request closed without merging**